server/resp/src/main/java/org/infinispan/server/resp/Resp3Handler.java

```diff
@@ -15,19 +20,19 @@
 import org.infinispan.security.AuthorizationPermission;
 import org.infinispan.server.resp.commands.Resp3Command;
 
-import java.util.Collection;
-import java.util.List;
-import java.util.concurrent.CompletionStage;
-import java.util.function.Consumer;
-import java.util.stream.Collectors;
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.ByteBufUtil;
+import io.netty.channel.ChannelHandlerContext;
 
-import static org.infinispan.server.resp.RespConstants.CRLF_STRING;
 public class Resp3Handler extends Resp3AuthHandler {
+   private static byte[] CRLF_BYTES = CRLF_STRING.getBytes();
    protected AdvancedCache<byte[], byte[]> ignorePreviousValueCache;
    protected EmbeddedMultimapListCache<byte[], byte[]> listMultimap;
    protected EmbeddedMultimapPairCache<byte[], byte[], byte[]> mapMultimap;
-   // Entry type for SetBucket needs proper hashcode, equals methods. Using WrappedByteArray
-   protected EmbeddedSetCache<byte[],WrappedByteArray> embeddedSetCache;
+
+   // Entry type for SetBucket needs proper hashcode, equals methods. Using
+   // WrappedByteArray
+   protected EmbeddedSetCache<byte[], WrappedByteArray> embeddedSetCache;
```

Comment:
We can update it later to work as the sorted set and support the use of `byte[]`.

Reply:
if not need I would stay with WBA, but yeah let's see later